lib/ansible/module_utils/fortios.py

```diff
@@ -0,0 +1,51 @@
+#
+# (c) 2017 Benjamin Jolivot, <bjolivot@gmail.com>
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
```

Comment:
Code in module_utils must not be GPL. BSD is prefered (also see the other scripts) or may even be a requirement, but I am not sure what other BSD like licenses are accepted.